tc/app/helpers/insert_string.help

```diff
@@ -0,0 +1,5 @@
+
+tcctl helpers insert_string --insert-string "STRING1" --file-name "FILE"
+
+--insert-string      "the string to insert" [REQUIRE]
+--file-name          "file name"  [REQUIRE]
```

Comment:
Require:
--insert-string      "the string to insert"
--file-name          "file name" 

Optional:
--optianal_key

It's better